telos-frontend/.prettierrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Looks like we can add more prettier rules here in the future if we find we need to, like setting tab width, trailing commas, etc.

Reply:
I think the rest of the settings are fine as default. We can always change in the future if needed.